docs/commerce/shipping_management/work_with_shipments.md

```diff
@@ -0,0 +1,65 @@
+---
+description: Work with shipments and view shipment details.
+edition: commerce
+---
+
+# Work with shipments
+
+In Ibexa DXP, on the **Shipments** screen, you can view a list of shipments and modify their statuses.
+You can access shipments for your own orders or all the shipments that exist in the system, depending on your permissions.
+
+![Shipment list](shipment_list.png)
+
+## View shipment status
+
+Each shipment has a status due to the stage it is currently at.
+
+Available statuses are:
+
+- Pending
+- Ready to ship
+- Shipped
+- Delivered
+- Cancelled
+
+![Shipment status](shipment_status.png)
+
+!!! note "Shipment statuses"
+
+    Shipment statuses visible in the **Status** filter field are defined in the [Shipment workflow]([[= developer_doc =]]/commerce/shipping_management/configure_shipment/#configure-shipment-workflow).
+
+To view shipment status:
+
+1\. In the left panel, go to **Commerce** -> **Shipments**.
+
+2\. Narrow down the list of displayed shipments in one of the following ways:
+
+- search for shipments by typing part of order ID or identifier in the search box
+- filter shipments by selecting one or more filters
+
+Available filters are:
+
+- Shipping method - method used for the shipment
+- Status - shipment status
+- Created - a range of dates between which the shipment was created
+- Updated - a range of dates between which shipment status has last changed
+
+![Shipment filters](shipment_filters.png)
+
+## View shipment details
+
+To view the details of a shipment, click its line in the shipment list.
+
+You can see an overview of the shipment's details on the shipment details screen. 
```

Comment:
```suggestion
On the shipment details screen, you can see an overview of the shipment's details. 
```
I'd state "where" first and "what" later. Also - this way you are connecting with the preceding sentence. The fact that the screen is visible is a consequence of clicking.

Reply:
@dabrt this sounds like "maslo maślane" :) because we repeat "shipment details" in one sentence.